db/migrate/201608152201_create_team_membership_invites_table.rb

```diff
@@ -0,0 +1,25 @@
+class CreateTeamMembershipInvitesTable < ActiveRecord::Migration
+  # rubocop:disable Metrics/MethodLength
+  def up
+    create_table :team_membership_invites do |t|
+      t.integer :team_id, null: false
+      t.integer :user_id, null: false
+      t.integer :inviter_id
+      t.boolean :refused, null: false, default: false
+      t.timestamps null: false
```

Comment:
Totally understandable, I'm still amazed at how fast you navigate through these changes. :D